A66-otel-stats.md

```diff
@@ -0,0 +1,542 @@
+# OpenTelemetry Metrics
+
+*   Author: Yash Tibrewal (@yashykt)
+*   Approver: Mark Roth (@markdroth)
+*   Status: In Review
+*   Implemented in: <language, ...>
+*   Last updated: Jul 20, 2023
+*   Discussion at: https://groups.google.com/g/grpc-io/c/po-deqYEQzE
+
+## Abstract
+
+Propose a metrics data model for gRPC OpenTelemetry metrics.
```

Comment:
Nit: this also specifies gRPC Instrumentation Component requirements and APIs

Reply:
I'm not sure what the offline discussion was here, but I think Zach is right: This gRFC does describe more than just the metric data model, and I think this line needs to be updated to reflect that.

Reply:
Offline, we discussed that it was fine as it is, and there weren't any strong opinions, so added it now.